pages/common/find.md

```diff
@@ -25,11 +25,11 @@
 
 - Run a command for each file (use `{}` within the command to access the filename):
 
-`find {{root_path}} -name '{{*.ext}}' -exec {{wc -l {} }}\;`
+`find {{root_path}} -name '{{*.ext}}' -exec {{wc -l {}}} \;`
```

Comment:
```suggestion
`find {{root_path}} -name '{{*.ext}}' -exec {{wc -l {} }} \;`
```
In my opinion, the space should stay. Otherwise clients could get confused on which of the braces belong to find's placeholder and which to tldr's.

Reply:
sounds like a good solution!